### ISSUE TITLE
 Michaelis-Gutmann bodies 

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5826]
**Preferred term label:**
 Michaelis-Gutmann bodies 

**Synonyms**

**Definition (free text, please give PubMed ID)**
eosinophilic granular macrophages with basophilic intracytoplasmic inclusions (Michaelis–Gutmann bodies) This b
the accumulation of tissue macrophages containing unique intracytoplasmic calcified inclusion bodies, known as Michaelis-Gutmann bodies. Various Gram-negative organisms have been reported to cause malakoplakia, most commonly Escherichia coli. The most common site of involvement is the urogenital tract followed by the gastrointestinal tract. Within the gastrointestinal tract, the rectum is the the most common site followed by the sigmoid colon.1
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5747605/

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7263748/

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
